File: trunk/sources/net.sf.j2s.ajax/store/net/sf/j2s/store/HTML5LocalStorage.java
package net.sf.j2s.store;


/**
 * 
 * @author Zhou Renjian (http://zhourenjian.com)
 *
 * July 10, 2010
 * 
 */
public class HTML5LocalStorage implements IStore {

	/**
	 * @j2sNative
	 * return localStorage.getItem (name);
	 */
	public String getProperty(String name) {
		return null;
	}

	public boolean isReady() {
		return true;
	}

	/**
	 * @j2sNative
	 * localStorage.setItem (name, value);
	 */
	public void setProperty(String name, String value) {
	}

}

File: trunk/incubator/net.sf.j2s.test.mauve/j2slib/net/sf/j2s/store/HTML5LocalStorage.js
$_J("net.sf.j2s.store");
$_L(["net.sf.j2s.store.IStore"],"net.sf.j2s.store.HTML5LocalStorage",null,function(){
c$=$_T(net.sf.j2s.store,"HTML5LocalStorage",null,net.sf.j2s.store.IStore);
$_V(c$,"getProperty",
function(name){
return localStorage.getItem(name);
},"~S");
$_V(c$,"isReady",
function(){
return true;
});
$_V(c$,"setProperty",
function(name,value){
localStorage.setItem(name,value);
},"~S,~S");
});
